packages/path_provider/path_provider_foundation/lib/path_provider_foundation.dart

```diff
@@ -68,4 +68,13 @@ class PathProviderFoundation extends PathProviderPlatform {
   Future<String?> getDownloadsPath() {
     return _pathProvider.getDirectoryPath(DirectoryType.downloads);
   }
+
+  /// Returns the path to the container of the specified App Group.
```

Comment:
Can you add a doc comment that this is only supported on iOS?

Reply:
Yep, done!